src/main/java/com/ngs/stash/externalhooks/hook/ExternalMergeCheckHook.java

```diff
@@ -155,7 +155,7 @@ public RepositoryHookResult preUpdate(PreRepositoryHookContext context, PullRequ
             }
         }
 
-        Repository repo = pr.getToRef().getRepository();
+        Repository repo = pr.getFromRef().getRepository();
```

Comment:
Isn't that going to change result of the following calls — [ExternalPreReceiveHook.java#L146-L150](https://github.com/reconquest/atlassian-external-hooks/blob/master/src/main/java/com/ngs/stash/externalhooks/hook/ExternalPreReceiveHook.java#L146-L150)?

Reply:
Oh, right. Didn't see that.

Reply:
We definitely need some test suite! 😄

Reply:
Updated to only change the repopath